deployer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin:v1.5.0-alpha.2
+FROM openshift/origin:v1.5.0
```

Comment:
ok, but we're not using the deployer in 1.5
next step should be to remove deployer/ and move anything needed into hack/